main.go

```diff
@@ -268,8 +269,11 @@ func InitFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&healthAddr, "health-addr", ":9440",
 		"The address the health endpoint binds to.")
 
-	fs.StringSliceVar(&additionalSyncMachineLabels, "additional-sync-machine-labels", []string{},
-		"List of regexes to select the additional set of labels to sync from the Machine to the Node. A label will be synced as long as it matches at least one of the regexes.")
+	fs.StringArrayVar(&additionalSyncMachineLabels, "additional-sync-machine-labels", []string{},
```

Comment:
```suggestion
	fs.StringSliceVar(&additionalSyncMachineLabels, "additional-sync-machine-labels", []string{},
```
(I changed this very recently on main)

(same below for annotations)